assets/src/styles/blocks/Covers/TakeActionBoxoutBottom.scss

```diff
@@ -74,7 +74,11 @@
     margin-bottom: 0;
     display: -webkit-box;
     -webkit-line-clamp: 2;
+    /* stylelint-disable property-no-vendor-prefix */
+    /*! autoprefixer: off */
```

Comment:
I had to add this because of an issue with autoprefixer. If we add the rule to prevent webkit prefixes, it complains about this. But if I then replace it with the unprefixed version stylelint complains about the property being unknown. See https://github.com/postcss/autoprefixer/issues/776#issuecomment-297734206

I used this fix so that we can apply the rule without risking this behaving differently. We probably need to check this anyway because it's a legacy property.